src/Microsoft.AspNet.JsonPatch/Operations/TypedOperation.cs

```diff
@@ -1,11 +1,12 @@
 ﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
 using Microsoft.AspNet.JsonPatch.Adapters;
 
 namespace Microsoft.AspNet.JsonPatch.Operations
 {
-    public class Operation<T> : Operation where T : class
+    public class Operation<TModel> : Operation where TModel : class
```

Comment:
Another note for cleanup - our guidelines say this file should be called `OperationOfT.cs`